src/auth/test/auth.controller.spec.ts

```diff
@@ -61,4 +63,21 @@ describe('AuthController', () => {
       });
     });
   });
+
+  describe('Change password', () => {
+    it('should change password', async () => {
+      //given
+      const request = {
+        newPassword: faker.internet.password(),
+      };
+      const userId = faker.number.int();
+      const spy = jest.spyOn(authService, 'changePassword');
+
+      //when
+      await authController.changePassword(userId, request);
+
+      //then
+      expect(authService.changePassword).toBeCalled();
+    });
```

Comment:
you could add more tests for it to be sure that password validation is correctly defined